### PR TITLE
Clarify the process for becoming a project reviewer/maintainer

### PR DESCRIPTION
**Pull request by rmondello**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)

### CODE REVIEW

**Comment by rmondello** [comment]
@bradcush @m33x @igor-makarov @jumoru @3NCRY9T3R @geoffcline @gobinathm @irew Would love your feedback on this.

**Comment by irew** [comment]
@rmondello looks good to me!

**Comment by 3NCRY9T3R** [comment]
> @bradcush @m33x @igor-makarov @jumoru @3NCRY9T3R @geoffcline @gobinathm @irew Would love your feedback on this.

Seems fine to me.
Good job.

File: README.md [hunk 1]
```diff
@@ -59,6 +59,19 @@ Please review [how to contribute](CONTRIBUTING.md) if you would like to submit a
...
 
-## Project Governance and Contact
+## Project Maintenance
 
-At the moment, merging a pull request requires approval by a project owner. If you or your organization is interested in becoming a project co-owner, ideally after contributing, please do get in touch! You can reach the project maintainers at Apple by [emailing password-manager-resources-maintainers@apple.com](mailto:password-manager-resources-maintainers@apple.com).
+Project maintenance involves, but is not limited to, adding clarity to incoming [issues](https://github.com/apple/password-manager-resources/issues) and reviewing pull requests. Project maintainers can approve and merge pull requests. Reviewing a pull request involves judging that a proposed contribution follows the project's guidelines, as described by the [guide to contributing](CONTRIBUTING.md). If you are interested in becoming a project maintainer, please [email the project maintainers at Apple by](mailto:password-manager-resources-maintainers@apple.com) with the following information:
```
Comment (bradcush) [inline README.md:64]: Small typo in "email the project maintainers at Apple by with the following information"

File: README.md [hunk 2]
```diff
@@ -59,6 +59,19 @@ Please review [how to contribute](CONTRIBUTING.md) if you would like to submit a
...
+
+* Your name
+* Your GitHub user name
+* Any organizations you're affiliated with that are related to password management, including professionally
+* Links to examples of pull requests submitted, review feedback given, and comments on issues that demonstrate why you'd be a good project project maintainer
```
Comment (bradcush) [inline README.md:69]: Typo in "project project"

**Comment by bradcush** [comment]
@rmondello Looks good, thanks for clarifying this!

**Comment by jumoru** [comment]
Brief and informative, looks good to me

**Comment by m33x** [comment]
Should we add a sentence about special cases? Like adult/hateful content, removing entries, adding security sensitive websites like paypal.com. Is this unrelated to becoming a project maintainer?

**Comment by rmondello** [comment]
> Should we add a sentence about special cases? Like adult/hateful content, removing entries, adding security sensitive websites like paypal.com. Is this unrelated to becoming a project maintainer?

This is important, but I think a separate issue from this. Do you know if we have an issue tracking this, @m33x?

**Comment by erynofwales** [comment]
Looks good! 🥳

**Comment by m33x** [comment]
> This is important, but I think a separate issue from this. Do you know if we have an issue tracking this, @m33x?

Sorry, I'm not aware of any.

**Pull request merged**